install_config/cluster_metrics.adoc

```diff
@@ -66,78 +66,28 @@ address of the node, then Heapster will fail to retrieve any metrics.
 ====
 endif::[]
 
-The components for cluster metrics must be deployed to the *openshift-infra*
-project. This allows xref:../dev_guide/pod_autoscaling.adoc#dev-guide-pod-autoscaling[horizontal pod
-autoscalers] to discover the Heapster service and use it to retrieve metrics
-that can be used for autoscaling.
-
-All of the following commands in this topic must be executed under the
-*openshift-infra* project. To switch to the *openshift-infra* project:
+An Ansible playbook is available to deploy cluster metrics. You should
+familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc[Advanced Installation and
```

Comment:
We need to have an anchor at the end of the path. In this case, we can use the ID at the top of the page you're pointing to. Also, looks like the topic is actually called Advanced Installation (not Advanced Installation and Configuration):
xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]